Bottom: 0e7d02acb0d11a6ed27d0d84965acdaefe149956
Top:    38cd18fddd9b6286458f57bf22bf97a557ac5a6a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-09-04 12:26:45 +0100

TEMPORARY: Comment out some DrawMark calls which might be out of context

Should fix this properly with notificatons to the GUI


---

diff --git a/src/action.c b/src/action.c
index 32e294c..edd79ba 100644
--- a/src/action.c
+++ b/src/action.c
@@ -4021,7 +4021,7 @@ ActionMarkCrosshair (int argc, char **argv, int x, int y)
     {
       if (Marked.status)
 	{
-	  DrawMark (true);
+//	  DrawMark (true);
 	  Marked.status = false;
 	}
       else
@@ -4029,16 +4029,16 @@ ActionMarkCrosshair (int argc, char **argv, int x, int y)
 	  Marked.status = true;
 	  Marked.X = Crosshair.X;
 	  Marked.Y = Crosshair.Y;
-	  DrawMark (false);
+//	  DrawMark (false);
 	}
     }
   else if (GetFunctionID (function) == F_Center)
     {
-      DrawMark (true);
+//      DrawMark (true);
       Marked.status = true;
       Marked.X = Crosshair.X;
       Marked.Y = Crosshair.Y;
-      DrawMark (false);
+//      DrawMark (false);
     }
   return 0;
 }
